src/View/mainpage/Toolbar.py

```diff
@@ -48,6 +48,8 @@ def __init__(self, action_handler: ActionHandler):
         self.addAction(self.action_handler.action_four_views)
```

Comment:
Just a few questions when testing the program:
1. When I choose an image set to be fused, the progress bar appears when loading the imageset, but no progress bar was shown when the fusing is in the process (apart from the printings in terminal, but nothing in GUI). Can we put some of the print statements in the progress bar instead to inform users that something is happening? I think statements like Creating Moving Model, Finished Creating Moving Model, Fusing Images would be useful, since the waiting is kind of long.


2. I can see that you allow user to open a fused imageset with rtstruct, but the warning that says cannot fuse with rtstruct is still there, please remove it.

3. When i try fusing two imagesets that have rtstruct to test the structure tab in image fusion, the program crashed and I received this error:
```
<class 'KeyError'> 204 <traceback object at 0x7f28e57c4900>
Traceback (most recent call last):
  File "/home/linh/OnkoDICOM/src/Controller/GUIController.py", line 151, in update_image_fusion_ui
    read_images_for_fusion()
  File "/home/linh/OnkoDICOM/src/Model/MovingModel.py", line 165, in read_images_for_fusion
    color_axial, color_sagittal, color_coronal = create_fused_model(orig_image, new_image)
  File "/home/linh/OnkoDICOM/src/Model/ImageFusion.py", line 295, in create_fused_model
    write_transform_to_dcm(affine_matrix)
  File "/home/linh/OnkoDICOM/src/Model/ImageFusion.py", line 246, in write_transform_to_dcm
    moving_dict_container.dataset[x].SOPClassUID
KeyError: 204
```

Reply:
1. We recently re-factored the fusion code to handle a fatal error with opening a new patient and the image fusion progress window was removed for convenience. We have a team member working on implementing it
2. We'll remove it
3. We'll investigate

Reply:
3. Upon investigation, could I possibly obtain your patient - images and the procedure on how to replicate this issue?

Reply:
@peteqian With the newest code, instead of the error above i got the message below:
```
ERROR:root:Traceback (most recent call last):
  File "/home/linh/OnkoDICOM/src/Model/Worker.py", line 77, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/home/linh/OnkoDICOM/src/View/ImageLoader.py", line 74, in load
    self.parent_window.signal_advise_calc_dvh.connect(self.update_calc_dvh)
AttributeError: 'ProgressWindow' object has no attribute 'signal_advise_calc_dvh'
Traceback (most recent call last):
  File "/home/linh/OnkoDICOM/src/Model/Worker.py", line 77, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/home/linh/OnkoDICOM/src/View/ImageLoader.py", line 74, in load
    self.parent_window.signal_advise_calc_dvh.connect(self.update_calc_dvh)
AttributeError: 'ProgressWindow' object has no attribute 'signal_advise_calc_dvh'
```

To replicate the message above:
- Open Onko and choose the SHL-CT CHEST dataset
- Choose Image fusion button, then choose the Thorax dataset.

Reply:
![image](https://user-images.githubusercontent.com/60642461/132211521-d396679f-65cf-4d9f-9a47-1c11318f4c2a.png)
![image](https://user-images.githubusercontent.com/60642461/132211653-6b9ac35a-c692-4491-9f9f-390cd7db8b6e.png)

Cannot replicate the issue

Reply:
When I selected the Thorax dataset for image fusion, I selected the whole study (including RTStruct, RTdose and RTplan), hence the error. I did see the warning saying that it cannot fuse with RTDose file, but if that's the case, it should not allow user to continue?